### PR TITLE
Add post announcing new members

### PR DESCRIPTION
**Pull request by kimadactyl**
So I added the logos to the theme and bumped the version number, I've not been able to test it properly tho because I can't work out how to get a dev setup up for that.

The config will also need applying to the data repo just run out of spoons for today and wanna get this up tomorrow am!

### CODE REVIEW

File: _news/2021-07-23-welcome-no-borders-trans-safety-network.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+Resistance Lab began as a simple provocation: can technology be used to help fight state violence, especially police brutality? 
+
+Beginning in 2018, representatives from Northern Police Monitoring Project and Geeks for Social Change facilitated a series of discussions between activists, campaigners, researchers, youth workers, academics and local residents. Over the first year our coalition grew to include both many individual members and group membership by Race, Roots and Resistance; The African Pot project; Kids of Colour; and Sites of Resistance.
+
+In August last year, we published our inaugural report [_A Growing Threat to Life: Taser usage by Greater Manchester Police_](/our-work/gmp-taser-update/index.html), which was then followed by a [report into the false claims made by GMP this year](/our-work/gmp-taser-update/index.html), and a [panel discussion on the use of Tasers nationally](/news/2021-02-25-no-more-tasers-panel.html). 
+
+Through this work we've been working closely with representatives from No Borders Manchester and Trans Safety Network who we are now delighted to announce are now formal group members of the Resistance Lab collective.xec
```
Comment (WheresAlice) [inline _news/2021-07-23-welcome-no-borders-trans-safety-network.md:15]: what's with the `.xec` on the end?

File: _news/2021-07-23-welcome-no-borders-trans-safety-network.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## No Borders Manchester
+
+No Borders Manchester are an activist group which aims to challenge border regimes in the UK and across Europe. To do this we conduct a wide range of activities to bring people together in resistance. 
+
+These include campaigns to raise awareness and challenge the Home Office on issues like deportation and #Stoptheplane; English classes both in person and online; and the Signing Support Network which supports people going to 'sign' at Dallas Court Reporting centre. Resistance Lab was the impetus for [the imok bot we launched to support the Signing Support Network back in May](https://gfsc.studio/2021/05/12/imok-is-launched.html). No Borders Manchester also run a community football group, conversation classes, publish a No Borders zine, and launched the day-long No Borders Festival last year. 
+
+[https://nobordersmcr.com/](https://nobordersmcr.com/){:.btn}
```
Comment (WheresAlice) [inline _news/2021-07-23-welcome-no-borders-trans-safety-network.md:23]: The `{:.btn}` is making it render weirdly, with a trailing `>` on the end

Additionally, maybe remove the trailing slash

Screenshot 2021-07-22 at 22 39 37

**Review by WheresAlice — CHANGES_REQUESTED** [review]

File: _news/2021-07-23-welcome-no-borders-trans-safety-network.md (hunk 1 above)
Comment (kimadactyl) [reply _news/2021-07-23-welcome-no-borders-trans-safety-network.md:15]: Ugh cut and pasted through a few too many editors!

File: _news/2021-07-23-welcome-no-borders-trans-safety-network.md (hunk 2 above)
Comment (kimadactyl) [reply _news/2021-07-23-welcome-no-borders-trans-safety-network.md:23]: The > is deliberate and meant to make it look more button-y (its in use in other places in the site too)

Happy to remove them tho agree it does look a bit weird after a link

**Review by WheresAlice — APPROVED** [review]

**Pull request merged**